systest/integration2/multitenancy_test.go

```diff
@@ -0,0 +1,136 @@
+//go:build integration2
+
+/*
+ * Copyright 2024 Dgraph Labs, Inc. and Contributors *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package main
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/dgraph-io/dgo/v240/protos/api"
+	"github.com/dgraph-io/dgraph/v24/dgraphapi"
+	"github.com/dgraph-io/dgraph/v24/dgraphtest"
+	"github.com/dgraph-io/dgraph/v24/x"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	gc *dgraphapi.GrpcClient
+	hc *dgraphapi.HTTPClient
+)
+
+func addData(t *testing.T, gc *dgraphapi.GrpcClient) {
+	require.NoError(t, gc.SetupSchema(`name: string  .`))
+
+	rdfs := `
+	_:a <name> "alice" .
+	_:b <name> "bob" .
+	_:c <name> "sagar" .
+	_:d <name> "ajay" .`
+	_, err := gc.Mutate(&api.Mutation{SetNquads: []byte(rdfs), CommitNow: true})
+	require.NoError(t, err)
+}
+
+func TestRunAllMultiTenancy(t *testing.T) {
```

Comment:
Is it possible to write an integration test instead of integration2?

Reply:
It is possible, but since we are moving from the existing test framework, I wrote these tests under the tag integration2.

Reply:
We are thinking about moving compose tests to use dgraphtest, not planning to move integration tests to integration2 from what I remember.